monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -165,11 +166,16 @@ object Rx {
   final case class Zip     [A, B]     (self: Rx[A], other: Rx[B])                   extends Rx[(A, B)]
   final case class DropRep [A]        (self: Rx[A])                                 extends Rx[A]
   final case class Merge   [A, B >: A](self: Rx[A], other: Rx[B])                   extends Rx[B]
+  final case class MergeN  [A, B >: A](self: List[Rx[A]])                           extends Rx[B]
```

Comment:
I would make it an explicit non empty list:

```scala
final case class class MergeN   [A, B >: A](self: Rx[A], other: List[Rx[A]]) extends Rx[B]
```

Reply:
But what is `self` supposed to be here?

Reply:
```
rxa.mergeN(rxb, rxc, rxd)
↑ this guy, "rxa"
```

Reply:
I thought so, but that approach looks a bit awkward to me at first glance. It will typically require the construction of some dummy Rx for `rxa`.